<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TalkbackSettingsForm</class>
 <widget class="QWidget" name="TalkbackSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="jtSelfLoopbackBox">
     <property name="text">
      <string>JackTrip Self Loopback</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="redundancyLabel">
       <property name="text">
        <string>Redundancy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="redundancyChoice">
       <property name="toolTip">
        <string>One fewer than the number of consecutive dropped packets allowed</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="encryptEnabledBox">
     <property name="text">
      <string>Enable Encryption</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>136</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="startJackTripButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start JackTrip</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="JackParameterForm" name="jackServer" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="Compressor" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JackParameterForm</class>
   <extends>QWidget</extends>
   <header>jackparameterform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Compressor</class>
   <extends>QWidget</extends>
   <header>compressor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
